src/test/java/dividing/DivideMethodTest.java

```diff
@@ -0,0 +1,34 @@
+package dividing;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please don’t use wildcard imports (*). It’s suggested to import only what is necessary 
Btw you can configure it in idea, and it won’t add wildcard imports by default